### ISSUE TITLE
Continuous scrolling of months

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2276]
Originally reported on Google Code with ID 2011

```
Hi arshaw
The full calendar is very useful and simple to configure , but i feel it miss one important
feature . that is scrollable Month view, when the user use scroll button or press down
button , it must scroll one week only ie in single view more than 1 month visible in
the screen , (Like android calendar)  refer the attachment for same
```

Reported by `ngl.satheesh` on 2013-10-04 12:37:19

<hr>
- _Attachment: [Full calander.JPG](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-2011/comment-0/Full calander.JPG)_

Imported with **22** stars.

**Comment by foodie2** [comment]
If you turn the scrollwheel on the page you are reading now, the page scrolls smoothly, it does not jump one screenful at a time. Why does Google Calendar not do the same? This seems to be what the OP, along with thousands of others, was asking. So why is the above marked 'closed'? (Please forgive newbie-like rant)

**Comment by espen** [comment]
1) This issue is not closed at all. Scroll up and you will see that it is in fact open.
2) "along with thousands of others" oh please.

**Comment by woodburn8** [comment]
Would really love to see this feature too.

**Comment by espen** [comment]
@woodburn8 then use the GitHub reaction feature so your vote will be counted. See more info at http://fullcalendar.io/wiki/Requesting-Features/

**Comment by georgehrke** [comment]
Can we from @Nextcloud help with this? :)

**Comment by shyamal890** [comment]
Can someone please update us on the status of this issue?

**Comment by Todd1561** [comment]
I too have a project that would greatly benefit from this.  Or if there were a way to load more than the current 6 weeks of data that would help in the meantime.  Is that possible?

**Comment by dnyg** [comment]
Any updates on this yet?

**Comment by FrancoisDupont** [comment]
So many projects are waiting for this update !

**Comment by AllyMurray** [comment]
@FrancoisDupont

FC is currently being re-written in TypeScript with some huge improvemens. If this is really important to you now, the current source code is available for you to change!

**Comment by FrancoisDupont** [comment]
@AllyMurray Yeah thanks and I found this for a continuous month scrolling for a year, by creating a new custom view based on 'basic' view.
       `CustomYear: {
            type: 'basic',
            duration: { months: 12 },
            defaults: {
                fixedWeekCount: true
            }
        }`
For now, it's really better than nothing !

**Comment by yadkee** [comment]
Please, someone who knows the code can give some guidelines as to what would need to be changed to get this functionality? It's something I miss very much.

**Comment by ianp5a** [comment]
Finding time for an appointment over a few weeks is hard if that goes into another month. You have to flip back and forth repeatedly with the Nextcloud Calendar app. You can't get an overview on one screen. A serious UI issue. I only use PC and android calendars that get this right. 
Some calendars have a "4-Week" view, that show _any_ 4 weeks at once, regardless of month, that would make using the Calendar less of a pain..

**Comment by alexanderdd** [comment]
Do I understand correctly that this issue has been solved in v6.1 with "daygridyear-view"? https://fullcalendar.io/blog/2023/01/multimonth-and-more

**Comment by mstllc** [comment]
> Do I understand correctly that this issue has been solved in v6.1 with "daygridyear-view"? https://fullcalendar.io/blog/2023/01/multimonth-and-more

@alexanderdd This specific issue is talking about an infinitely/continuous scrolling of months, which is **not** solved yet.  v6.1 introduced that daygridyear view which is close, but only gives you a single year of months and does not scroll continuously.  when you render that new view, you have a vertically scrolling list of months for the current year (jan-dec of the current year, not feb-feb for example).

**Comment by zachbresler21** [comment]
Is there any hope of having this feature soon? It would be extremely useful

**Comment by ideesnoires** [comment]
At least an scrolledToEnd/Start Event for any given calendar view would be great. I built a solution for Timeline view that's using an eventlistener on the fc-scoller element which is not so nice.

**Comment by henningmu** [comment]
@arshaw we're big fans of fullcalendar over at Doist. We would love to have this feature in our app and are willing to fund its development. Please feel free to reach out to me at `henning@doist.com` to discuss details 🙌

**Comment by hfauq** [comment]
+1 Would love to see continuous scrolling implemented. 

This is what we mean with this (see the Cron app):

https://github.com/fullcalendar/fullcalendar/assets/19209269/3171c450-f458-4d10-bf59-da3923afcf7d

**Comment by mike-koala-bear** [comment]
+1 Would love to see this feature!

**Comment by seanahrens** [comment]
@joshua-martius you may be right. although a lot of the language in this ticket uses "continuous" as the term so I'm not sure.

It appears to me that one can use dayGridContinuous as a workaround more or less for both.

`        views={{
          dayGridContinuous: {
            type: 'dayGrid',
            duration: { months: 60 },
            titleFormat: { month: 'short', day: 'numeric', year: 'numeric' }
          }
        }}`

My work with dayGridContinuous allows me to go to a defined number of months into the future (be it 3, 12, or 60). So not infinite, but if you manage the scrolling yourself in the containing component, it vaguely appears to act "infinite" (up until a horizon/end-date you set). Obviously the limitations of this workaround are 1) that horizon/end-date 2) performance, since presumably all the data is greedily loaded, instead of onScroll.

There is some documentation on it here: https://fullcalendar.io/docs/daygrid-view#year-view, but it's not entirely clear.
